src/components/AddPlaidBankAccount.js

```diff
@@ -249,7 +249,7 @@ function AddPlaidBankAccount({
                     height={iconSize}
                     width={iconSize}
                     additionalStyles={iconStyles}
-                    fill={theme.icon}
+                    displayInDefaultIconColor
```

Comment:
We don't require to pass displayInDefaultIconColor here as it is only used inside MenuItem.

Reply:
MenuItem has default fill color if `displayInDefaultIconColor` is not specified

Reply:
But this is simply an Icon not a MenuItem.

Reply:
@shubham1206agra  Do you agree that `displayInDefaultIconColor` is irrelevant here?

Reply:
Oh right. Overlooked it.